### PR TITLE
Do not set manually sourced on Cost estimate lines when LRS is Other or ...

### PR DESCRIPTION
**Pull request by yvaucher**
...WH Dispatch

### CODE REVIEW

**Comment by gurneyalex** [comment]
:+1:

File: logistic_requisition/wizard/cost_estimate.py [hunk 1]
```diff
@@ -104,6 +104,19 @@ def _get_route_drop_shipping(self):
         self.env['ir.model.data'].xmlid_to_res_id(ref)
 
     @api.model
+    def _is_manually_source(self, sourcing):
```
Comment (lepistone) [inline logistic_requisition/wizard/cost_estimate.py:107]: This method has a source as parameter, so I would use `@api.multi` and work on `self`. That way we can later say `source.is_manually_source()` which sounds logical.

Also, maybe we can add a `d` to the method name.

File: logistic_requisition/wizard/cost_estimate.py [hunk 2]
```diff
@@ -113,7 +126,7 @@ def _prepare_cost_estimate_line(self, sourcing):
                 'product_uom_qty': sourcing.proposed_qty,
                 'product_uom': sourcing.proposed_uom_id.id,
                 # line must be sourced
```
Comment (lepistone) [inline logistic_requisition/wizard/cost_estimate.py:128]: This comment is probably not relevant anymore, we can get rid of it IMO.

**Comment by lepistone** [comment]
With minor remarks, :+1: thanks.

**Comment by nbessi** [comment]
:+1:

**Comment by lepistone** [comment]
I had a few pending remarks, but nothing too bad. Thanks

**Pull request merged**